src/client/dfuse/SConscript

```diff
@@ -151,6 +151,9 @@ def configure_fuse(cenv):
                                'cache_readdir'):
         cenv.AppendUnique(CPPDEFINES=['-DHAVE_CACHE_READDIR=1'])
 
+    if check.CheckHeader('fuse3/fuse_log.h'):
+        cenv.AppendUnique(CPPDEFINES=['-DHAVE_FUSE_LOG=1'])
```

Comment:
Didn't realize you could do this without defining the 0 case but my local test doesn't issue a warning.   Maybe there was some compiler in my past that complained about an undefined macro....or maybe I'm just dreaming.